documentation/getting_started_guide/GETTING_STARTED.ipynb

```diff
@@ -360,33 +360,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### 5.2 Opening your dataset from a local directory over HTTP\n",
+    "### 5.2 Open locally via CLI\n",
     "\n",
-    "To view a **locally converted dataset**, we'll also need to run a **local version** of the Timelapse Feature Explorer.\n",
+    "We've provided a simple command-line script to open a local dataset in TFE, `tfe-open`. It launches a local instance of the viewer and serves the dataset over HTTP.\n",
     "\n",
-    "> ⚠ If you've been following along in Google Colab, this is the point where you'll want to switch to your local machine. Rerun the previous steps locally or download the processed dataset into `colorizer-data/documentation/getting_started_guide/processed_dataset` and continue from here.\n",
+    "> ⚠ The `tfe-open` script will not work in Google Colab, so this step can only be run on a local machine.\n",
     "\n",
-    "<details>\n",
-    "<summary><b>[Why do we need to run a local instance of the viewer?]</b></summary>\n",
-    "\n",
-    "--- \n",
-    "\n",
-    "TFE only loads data over the HTTP(S) web protocols, rather than directly loading local files. We can get around this by running a local server to serve our local files over HTTP.\n",
-    "\n",
-    "However, our public build of TFE is served over HTTPS, a secure version of the HTTP protocol. For security reasons, HTTPS pages cannot load HTTP content. This means that the public version of TFE can only access other web content also hosted on HTTPS. \n",
-    "\n",
-    "It's difficult to set up an HTTPS server, so instead we can run TFE locally on HTTP! This will let it also access our local HTTP server.\n",
-    "\n",
-    "![A diagram. The public version of TFE (HTTPS) can load files from a remote file server (HTTPS) but can't load it from a local file server (HTTP). A local version of TFE (HTTP) can load from both remote file servers and local file servers (HTTP and HTTPS).](./assets/http-diagram.png)\n",
-    "\n",
-    "---\n",
-    "</details>\n",
```

Comment:
All of this description ended up being way too much technical detail! It's much simpler now.